oap-server/server-cluster-plugin/cluster-etcd-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/etcd/EtcdCoordinator.java

```diff
@@ -113,7 +114,13 @@ private void renew(EtcdClient client, String key, String json) {
             try {
                 client.refresh(key, KEY_TTL).send().get();
             } catch (Exception e) {
-
+                try {
+                    client.put(key, json).ttl(KEY_TTL)
+                        .setRetryPolicy(new RetryNTimes(1, 10))
+                        .send()
+                        .get();
+                } catch (Exception ee) {
```

Comment:
We'd better not swallow the exception, at least, log a warning message to notify users that the Etcd connection is unstable or something

Reply:
ok